src/main.rs

```diff
@@ -393,6 +482,62 @@ enum NorgASTFlat {
     },
 }
 
+fn detached_modifier_extensions(
+) -> impl Parser<NorgToken, Vec<DetachedModifierExtension>, Error = chumsky::error::Simple<NorgToken>>
+{
+    use NorgToken::*;
+
+    let detached_modifier_extension_tokens: Vec<NorgToken> =
+        "@#<>+= x_-!?".chars().map(norg_char).collect();
+
+    // a single extension without the surrounding `()` or delimiting `|`
+    // This parsing is not going to happen at the block level, it will happen later I guess.
+    let detached_modifier_extension = one_of(detached_modifier_extension_tokens)
```

Comment:
I think I'd prefer a more verbose `select!` invocation instead of this:
```suggestion
    let detached_modifier_extension_tokens = select! {
        Whitespace => (),
        Special('@' | '#' | '<' | ...) => (),
    };
    
    let detached_modifier_extension = detached_modifier_extension_tokens
```